docs/ref/models/querysets.txt

```diff
@@ -979,6 +979,16 @@ database.
     performance, since you have done a database query that you haven't used. So
     use this feature with caution!
 
+.. versionchanged:: 1.11
```

Comment:
Sorry, I'm struggling to understand that particular bit of instructions in the documentation. 

If I read the instructions correctly, you want me to move the `versionchanged` notes to the bottom of the `prefetch_related` section so that no formatting changes will occur upon removal. Is my understanding correct?